skada/_reweight.py

```diff
@@ -8,8 +8,9 @@
 
 import numpy as np
 from scipy.stats import multivariate_normal
+from scipy.optimize import minimize, LinearConstraint
```

Comment:
We will have  a conflict between antoines here (@antoinedemathelin VS @antoinecollas ) ;)